apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -245,21 +221,15 @@ var TopInstructions = React.createClass({
     ];
     const ttsUrl = this.props.ttsMarkdownInstructionsUrl;
     const videoData = this.props.levelVideos ? this.props.levelVideos[0] : [];
-    const logText = JSON.stringify({'AppType': this.props.app, 'ScriptName': this.props.scriptName,
-      'StagePosition': this.props.stagePosition, 'LevelPosition': this.props.levelPosition});
-
-    // If we are in the resources tab experiment, only display the help tab when
-    // are one or more videos
-    const resourcesTabDisplayTab = (experiments.isEnabled('resources_tab') || experiments.isEnabled('resourcesTab')) && this.props.levelVideos.length > 0;
 
+    // If we are in the additional resources experiment, only display the help tab
+    // when there are one or more videos or additional resource links.
```

Comment:
It looks like there is a mismatch between this comment and the logic below, because the logic will always show the help tab when in the additionalResources experiment.